src/wp-includes/class-wp-block-type.php

```diff
@@ -272,6 +272,45 @@ class WP_Block_Type {
 		'style',
 	);
 
+	/**
+	 * Experimental to stable block supports map.
+	 *
+	 * @since 6.8.0
+	 * @var array
+	 */
+	private $experimental_supports_map = array( '__experimentalBorder' => 'border' );
+
+	/**
+	 * Map of shared experimental support properties to stable.
+	 *
+	 * This map relates to experimental block support flags that are
+	 * across multiple different block supports.
+	 *
+	 * @since 6.8.0
+	 * @var array
+	 */
+	private $common_experimental_support_properties = array(
+		'__experimentalDefaultControls'   => 'defaultControls',
+		'__experimentalSkipSerialization' => 'skipSerialization',
+	);
+
+	/**
+	 * Experimental to stable block supports properties map.
+	 *
+	 * @since 6.8.0
+	 * @var array
+	 */
+	private $experimental_support_properties = array(
```

Comment:
Oh I just realized, this isn't being used anywhere, and it doesn't match Gutenberg trunk. 

https://github.com/WordPress/gutenberg/blob/trunk/lib/compat/wordpress-6.8/blocks.php#L28C2-L28C34

Reply:
~Oh I just realized, this isn't being used anywhere~ 

Ignore me.